### PR TITLE
Issue-49 fix - add prefix to `owner` attribute

### PR DESCRIPTION
**Pull request by RuslanKhabibullin**
https://github.com/DmitryTsepelev/ar_lazy_preload/issues/49

### CODE REVIEW

**Comment by PikachuEXE** [comment]
Would be good to have test cases that reproduce what's fixed

**Comment by RuslanKhabibullin** [comment]
@PikachuEXE I've added a simple spec that checks that there are no exceptions for entities with `owner` enum value. An error can be reproduced with the next `Gemfile.local` file + old `AssociationRelation` class (with `owner` delegation without prefix):

```
source 'https://rubygems.org'

gem "sqlite3", "~> 1.4"
gem "rails", "~> 6.1.1"
```

File: spec/ar_lazy_preload/active_record/association_relation_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+require 'spec_helper'
+
+describe ArLazyPreload::AssociationRelation do
+  let(:user) do
+    user = create :user
+    club = create :club
+    create :club_member, club: club, user: user, role: :owner
+    user
+  end
```
Comment (DmitryTsepelev) [inline spec/ar_lazy_preload/active_record/association_relation_spec.rb:11]:
```suggestion
  let(:club) { create(:club) }
  let(:user) { create(:user) }
  
  before do
    create :club_member, club: club, user: user, role: :owner
  end
```

File: spec/ar_lazy_preload/active_record/association_relation_spec.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  it 'not raise any error when `owner` method already defined in model' do
+    expect { p user.club_memberships }.not_to raise_error
```
Comment (DmitryTsepelev) [inline spec/ar_lazy_preload/active_record/association_relation_spec.rb:14]: Can we replace `user.club_memberships` with `user.club_memberships.load` ?

**Review by DmitryTsepelev — CHANGES_REQUESTED** [review]
Hi @RuslanKhabibullin, thank you for the PR!

Let's fix RuboCop, couple of questions in below, please add yourself to the changelog and we'll be good to go.

**Comment by RuslanKhabibullin** [comment]
@DmitryTsepelev thank you for your review! Seems like done.

**Review by DmitryTsepelev — APPROVED** [review]

**Pull request merged**